locales/en/kafka.json

```diff
@@ -73,7 +73,7 @@
     "reset_offset": "Reset offset",
     "reset_offset_accept": "I understand that by resetting the offset position, I risk clients skipping or duplicating messages.",
     "reset_offset_connected_alert_body": "One or more members of the consumer group are currently connected to a topic. All members of a consumer group must be disconnected before resetting the offset.",
-    "reset_offset_connected_alert_title": "The offset for this consumer group cannot be reset at this time",
+    "reset_offset_connected_alert_title": "The offset for this consumer group caénnot be reset at this time",
```

Comment:
Is caennot a typo?

Reply:
yep, not sure how it did end there. Thanks for checking!